scripts/qa_crowbarsetup.sh

```diff
@@ -415,7 +415,7 @@ function add_ha_repo()
         fi
         # Note no zypper alias parameter here since we don't want to
         # zypper addrepo on the admin node.
-        add_mount "$repo" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
+        add_mount "$repo/sle-11-x86_64" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
```

Comment:
> No, this is for HAE not SES!

Ah. Sorry. Then looks good to me. But I guess there's a bug in the storage function then.